roles/housekeeping/tasks/users_groups_dirs.yml

```diff
@@ -1,24 +1,24 @@
 ---
-- name: chown /opt
-  file: owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} state=directory path=/opt
+- name: chown {{ project_base }}
+  file: owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} state=directory path={{ project_base }}
 
 - name: create install directory
   file: owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} state=directory path={{ install_path }}
 
-- name: create cap group
-  group: name={{ capistrano_group }} state=present
+- name: create deploy group
+  group: name={{ deploy_group }} state=present
 
-- name: create cap user
-  user: name={{ capistrano_user }} group={{ capistrano_group }} shell=/bin/bash createhome=yes state=present
+- name: create deploy user
+  user: name={{ deploy_user }} group={{ deploy_group }} shell=/bin/bash createhome=yes state=present
 
-- name: create cap directories
-  file: path={{ item }} state=directory owner={{ capistrano_user }} group={{ capistrano_group }} 
+- name: create project directories
```

Comment:
I'd change this from "project directories" to "deployment directories" to make the connection to a deployment tool obvious.